packages/multichain-network-controller/src/utils.ts

```diff
@@ -33,3 +42,47 @@ export function checkIfSupportedCaipChainId(
   // Check if the chain id is supported
   return Object.keys(AVAILABLE_MULTICHAIN_NETWORK_CONFIGURATIONS).includes(id);
 }
+
+/**
+ * Updates a network configuration to the format used by the MultichainNetworkController.
+ * This method is exclusive for EVM networks with hex identifiers from the NetworkController.
+ *
+ * @param network - The network configuration to update.
+ * @returns The updated network configuration.
+ */
+export const updateNetworkConfiguration = (
```

Comment:
Perhaps minor since we can always change it later, but should this be called `toMultichainNetworkConfiguration`? Saying that it "updates" the network configuration makes it sound like it changes it, when it in fact converts from one format to another.

Reply:
good point, can be done now

Reply:
https://github.com/MetaMask/core/pull/5315/commits/832e57e42e523c1261bdc8faf6a5e148588a5acf